### ISSUE TITLE
Erythrocyte casts

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5046]
Definition:Presence of red blood cell casts
Modified definition: Presence of erythrocyte casts (cylindrical structures produced by the kidney in certain disease states) in the urine.

**Comment by pnrobinson** [comment]
Synonym with Erythrocyte cylindruria   https://hpo.jax.org/app/browse/term/HP:0012617
I will change the primary label to Erythrocyte casts, OK?

@avizrosenberg
@beckysteck
@yongqunh

**Comment by yongqunh** [comment]
Sounds good to me.

**Comment by avizrosenberg** [comment]
Erythrocyte cylinduria reflects a urine microscopic definition (synonymous with red blood cell casts). Red blood cell or erythrocyte casts in the KPMP histopathology context would be restricted to their presence on tissue microscopic examination.
